src/lib.rs

```diff
@@ -325,10 +325,10 @@ pub trait Rng: RngCore {
     /// `low` and exclusive of `high`.
     ///
     /// This is a convenience wrapper around
-    /// `distributions::Range`. If this function will be called
-    /// repeatedly with the same arguments, one should use `Range`, as
+    /// `distributions::Uniform`. If this function will be called
+    /// repeatedly with the same arguments, one should use `Uniform`, as
     /// that will amortize the computations that allow for perfect
-    /// uniformity, as they only happen when constructing the `Range`.
+    /// uniformity, as they only happen when constructing the `Uniform`.
```

Comment:
Can we mention "range" one or two times in this comment? I don't want to see no mention at all to this common term any more.

Reply:
It is already mentioned in the name and the fist sentence.

Reply:
I agree with @pitdicker actually that a couple of things could be changed:

- "wrapper around `distributions::Uniform::sample_single`"
- "when constructing the `Uniform` range."